### DISCUSSION TITLE
Does Deepgram use data submitted to Whisper algorithms to train models?

### DISCUSSION THREAD

**Discussion by maboa** [deepgram/community#599] [General help] [answered]
Apparently Deepgram can uses all data for model training purposes, I just wanted to check that this is true even when submitting data to the Whisper models. Thanks!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @maboa, yes, our models all use the same terms of service. Linking the relevant pages for reference:
[Data / Security / Privacy FAQ](https://help.deepgram.com/hc/en-us/articles/6126293557399-Data-Security-Privacy-FAQ)
[Deepgram Privacy Policy](https://deepgram.com/privacy)
[Terms of Service](https://deepgram.com/terms)

Depending on your expected scale, we do offer an [on-prem option](https://developers.deepgram.com/docs/on-prem-introduction), where we provide you with our models, which you host on your servers (cloud or bare-metal), and your data does not leave your environment. This is popular with customers who have strict data requirements. It is only available with an [Enterprise](https://deepgram.com/pricing) plan that starts at a committed spend of $10k/year.

**Reply by maboa** [reply]
OK. Thanks for the prompt clarification @jkroll-deepgram Much appreciated.
